src/Ledger/Conway/Specification/Enact/Properties/Computational.agda

```diff
@@ -13,6 +13,21 @@ open EnactState
 
 open Computational ⦃...⦄
 
+module _ {Γ : EnactEnv} {eSt : EnactState} {ga : GovAction} where
+
+  -- TODO: Make this follow from computationalness or viceversa
+  ENACT-deterministic : ∀ {eSt' eSt''}
+                      → Γ ⊢ eSt ⇀⦇ ga ,ENACT⦈ eSt'
+                      → Γ ⊢ eSt ⇀⦇ ga ,ENACT⦈ eSt''
+                      → eSt' ≡ eSt''
+  ENACT-deterministic Enact-NoConf Enact-NoConf = refl
+  ENACT-deterministic (Enact-UpdComm _) (Enact-UpdComm _) = refl
+  ENACT-deterministic Enact-NewConst Enact-NewConst = refl
+  ENACT-deterministic Enact-HF Enact-HF = refl
+  ENACT-deterministic Enact-PParams Enact-PParams = refl
+  ENACT-deterministic (Enact-Wdrl _) (Enact-Wdrl _) = refl
+  ENACT-deterministic Enact-Info Enact-Info = refl
```

Comment:
You could give it a name which (imho) more precisely describes the what the theorem says; e.g., `ENACT-is-a-function` or (in Andre's terminology) `ENACT-is-left-unique`... but if you prefer to stick with `ENACT-deterministic`, that's fine; it's not wrong.